control_plane/src/safekeeper.rs

```diff
@@ -117,14 +117,14 @@ impl SafekeeperNode {
         let listen_pg = format!("127.0.0.1:{}", self.conf.pg_port);
         let listen_http = format!("127.0.0.1:{}", self.conf.http_port);
 
-        let mut cmd = Command::new(self.env.safekeeper_bin()?);
-        fill_rust_env_vars(
-            cmd.args(&["-D", self.datadir_path().to_str().unwrap()])
-                .args(&["--id", self.id.to_string().as_ref()])
-                .args(&["--listen-pg", &listen_pg])
-                .args(&["--listen-http", &listen_http])
-                .arg("--daemonize"),
-        );
+        let mut command = Command::new(self.env.safekeeper_bin()?);
+        let cmd = command
+            .args(&["-D", self.datadir_path().to_str().unwrap()])
+            .args(&["--id", self.id.to_string().as_ref()])
+            .args(&["--listen-pg", &listen_pg])
+            .args(&["--listen-http", &listen_http])
+            .arg("--daemonize");
+        let mut cmd = fill_rust_env_vars!(cmd);
```

Comment:
Let's get rid of `daemonize` for safekeeper too.

Reply:
Done, also removed daemonize from logging.